src/vm-vcpu/src/vm.rs

```diff
@@ -245,7 +249,9 @@ mod tests {
         // We're specifying in the VmState that we want to run one vcpu, but we do not create
         // any. This should return an error.
         let (mut vm, _guest_memory) = default_vm(1).unwrap();
-        assert!(matches!(vm.run(), Err(Error::RunVcpus(e)) if e.kind() == ErrorKind::InvalidInput));
+        assert!(
+            matches!(vm.run(GuestAddress(0x10000000)), Err(Error::RunVcpus(e)) if e.kind() == ErrorKind::InvalidInput)
```

Comment:
nit (how did clippy miss this?): `0x1000_0000`

Reply:
I don't think clippy runs on tests...

Reply:
Done.